crates/red_knot/README.md

```diff
@@ -0,0 +1,27 @@
+# Red Knot
+
+Red Knot is an extremely fast type checker.
```

Comment:
well... we _hope_ it will be. I'm not sure I'm comfortable saying that it is "extremely fast" relative to other type checkers _yet_, because there's still so much that they do that we don't.

Reply:
Heh, fair enough, I suppose. I guess we're not displaying comparative benchmarks yet, anyway. I'd like to check other members of the team are also okay with this, however.

Reply:
I'm fine with it and it's easy to change if it turns out that red knot is slow (I think it's about 30x faster on my machine when comparing black)